qrenderdoc/Windows/TextureViewer.cpp

```diff
@@ -741,7 +846,21 @@ TextureViewer::TextureViewer(ICaptureContext &ctx, QWidget *parent)
 
   ui->textureList->setModel(new TextureListItemModel(m_Ctx, this));
   ui->textureList->setItemDelegate(new TextureListItemDelegate(ui->textureList));
+  ui->textureList->viewport()->installEventFilter(this);
   ui->textureList->viewport()->setAttribute(Qt::WA_Hover);
+  ui->textureList->setMouseTracking(true);
+
+  {
+    RDHeaderView *header = new RDHeaderView(Qt::Horizontal, this);
+    ui->textureList->setHeader(header);
+
+    header->setColumnStretchHints({-5, -4, -3, -2, -1, 1});
```

Comment:
This isn't well documented but the way the stretch hints are implemented all negative values are treated the same. The logic is that columns with a negative value only get just enough space to fit their contents, and any remaining extra space is assigned to columns with non-negative values, with the actual value indicating the proportion (i.e. if they're all 1 then the extra space is split evenly, if one of them was 2 it would get twice as much extra space as a column with 1).

So this is fine, it determines a layout where only the last column gets extra space which is a common pattern but it'd be clearer to have the other columns all be -1.

ninja edit: Actually I think the first name column should be 1 and the others all -1, because they're logical columns not visual columns. Otherwise if you re-order the columns to put format in the middle somewhere it looks really weird being very wide and crushing the others on the right.